NMS-15425: Update dual write plugin docs (#5788)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/admin/system-properties/introduction.adoc

The priority for Java system properties is as follows:

. Those set via the Java command line (for example, in `opennms.conf` via `ADDITIONAL_MANAGER_OPTIONS`).
. `opennms.properties.d/*.properties` +
Files in this directory are loaded alphabetically.
. `opennms.properties`
. `libraries.properties`
. `rrd-configuration.properties`
. `bootstrap.properties`

TIP: To avoid conflicts with customized configurations, you can add all custom properties to one or more files in `$\{OPENNMS_HOME}/etc/opennms.properties.d/`.
We recommend that you do not modify default OpenNMS properties.

> Create dedicated files with your customized properties in `opennms.properties.d/`.

Reviewer: ```suggestion
Instead, create dedicated files with your customized properties in `opennms.properties.d/`.
```

The priority for Java system properties is as follows:

> . Those set via the _Java_ command line i.e., in `opennms.conf` via `ADDITIONAL_MANAGER_OPTIONS`

Reviewer: ```suggestion
. Those set via the Java command line (for example, in `opennms.conf` via `ADDITIONAL_MANAGER_OPTIONS`).
```

. `opennms.properties.d/*.properties` +
Files in this directory are loaded alphabetically.
. `opennms.properties`
. `libraries.properties`
. `rrd-configuration.properties`
. `bootstrap.properties`

TIP: To avoid conflicts with customized configurations, you can add all custom properties to one or more files in `$\{OPENNMS_HOME}/etc/opennms.properties.d/`.

> We recommend not to modify default OpenNMS properties.

Reviewer: ```suggestion
We recommend that you do not modify default OpenNMS properties.
```
